parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java

```diff
@@ -472,7 +488,13 @@ public static boolean getPageWriteChecksumEnabled(Configuration conf) {
     LOG.debug("Parquet properties are:\n{}", props);
 
     WriteContext fileWriteContext = writeSupport.init(conf);
-    
+
+    MessageType schema = fileWriteContext.getSchema();
+    HashSet<Type.ID> ids = new HashSet<>();
```

Comment:
this variable is not used outside `checkDuplicateId`; is there a way to hide it?

Reply:
Fixed. Thanks!